----------------------------------------<br>Logging session at compiled time : 19:14:51 Nov 22 2022<br><br>In file main.cpp, function main, line 12, Stack named lol was dumped : <br>	Errors : <br>		[POISONED_ERRCOD     ]<br>		[WRONG_HASH          ]<br>	hash = 2998378365 (ok)<br>	canL = 0xDEADBEEF (ok)<br>	canR = 0xD34DB33F (ok)<br>	size = 5 (ok)<br>	capacity = 8 (ok)<br>	dataCanL = 0xDEADBEEF (ok)<br>	dataCanR = 0xD34DB33F (ok)<br>	lol.stack : <br>		lol.data[0] : 1;<br>		lol.data[1] : 2;<br>		lol.data[2] : 3;<br>		lol.data[3] : 3;<br>		lol.data[4] : 3;<br>		lol.data[5] : 0;<br>^^(POISONED)^^<br>		lol.data[6] : 0;<br>^^(POISONED)^^<br>		lol.data[7] : 0;<br>^^(POISONED)^^<br>	End of stack<br>End of dump <br>